packages/embark-web3/src/index.js

```diff
@@ -59,6 +59,7 @@ class EmbarkWeb3 {
       await this.events.request2("runcode:register", contract.className, result);
       cb();
     } catch (err) {
+      console.dir(err);
```

Comment:
Either remove or use the logger if maybe you think it's a useful error to show. Could be using `logger.debug`?